### ISSUE TITLE
RTL Issue in Indicator

### ISSUE THREAD

**Issue by Salmancs43** [uias/Tabman#443]
Thanks for amazing Library. 
I have an issue related to RTL. When I change language from English to Arabic then the indicator remains at the non selected part as shown in video. You can see "Pending" option is selected but indicator shown behind the "Cancelled" button. When I drag the controller a little then the indicator working fine. Can you please help me with that.
Here is the code I write
<img width="812" alt="Screenshot 2019-07-21 at 8 46 01 PM" src="https://user-images.githubusercontent.com/23050315/61593409-9580dc00-abf8-11e9-8067-144417de0eee.png">

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/23050315/61593403-7e41ee80-abf8-11e9-917b-88b64ed1e222.gif)

Thanks

**Comment by msaps** [comment]
@Salmancs43 I can't seem to reproduce this with the Example project - could you maybe provide a demo that reproduces the issue?? Thanks!

**Comment by foratbahrani** [comment]
Hello, I have the same issue with RTL languages too. here's a demo:

![ezgif-1-27e21ce82b34](https://user-images.githubusercontent.com/46747287/63228569-7d9f7680-c1fd-11e9-8cf1-dabc601a66d0.gif)

and this is my code:

**Comment by zaidqattan** [comment]
> Hello, I have the same issue with RTL languages too. here's a demo:
> 
> ![ezgif-1-27e21ce82b34](https://user-images.githubusercontent.com/46747287/63228569-7d9f7680-c1fd-11e9-8cf1-dabc601a66d0.gif)
> 
> and this is my code:
> <img alt="Screen Shot 2019-08-18 at 21 18 44" width="906" src="https://user-images.githubusercontent.com/46747287/63228591-d1aa5b00-c1fd-11e9-9912-aca0f4e23c79.png">

Same Issue for me, any solution for this problem?

**Comment by zaidqattan** [comment]
@msaps

**Comment by Kashif-Rabbani** [comment]
Hi @msaps, I am facing the same problem in RTL mode. Can you please look into it? Many Thanks.

**Comment by msaps** [comment]
@Kashif-Rabbani @zaidqattan I still haven't been able to reproduce this unfortunately 😞 - can you try reproducing it in the Tabman example project? I've tried multiple times to reproduce this and can't quite seem to do it.

**Comment by Kashif-Rabbani** [comment]
@msaps Oh, that's very simple. I have the repo project setup. You just need to add this line 

`UIView.appearance().semanticContentAttribute = .forceRightToLeft` in file [TabPageViewController.swift](https://github.com/uias/Tabman/blob/master/Example/Tabman-Example/TabPageViewController.swift) at line 40. 

And you will see this behavior on launch as shown in the screenshot below: 

<img width="468" alt="image" src="https://user-images.githubusercontent.com/36715664/80278561-bc27f200-86f7-11ea-9e2f-18e8dce6eb3a.png">

Please let me know if you need any assistance from my side. 

Many Thanks.

**Comment by msaps** [comment]
@Kashif-Rabbani that's really strange - I've added that line at line 40:

```swift
override func viewDidLoad() {
        super.viewDidLoad()
        UIView.appearance().semanticContentAttribute = .forceRightToLeft
```

And it seems to still lay out fine with the same number of pages / device 🤔 

<img width="584" alt="Screenshot 2020-04-25 at 14 25 28" src="https://user-images.githubusercontent.com/3515448/80279897-9e12bf80-8700-11ea-9508-cec44a763bdc.png">

**Comment by Kashif-Rabbani** [comment]
@msaps which version of Tabman are you using?
I am using the one listed in the documentation of repo as follow: `pod 'Tabman', '~> 2.6'`

**Is it the latest one?**

**Comment by msaps** [comment]
@Kashif-Rabbani just noticed I had a Scheme option enabled to use RtL pseudo language - have managed to reproduce now 👍 

The latest is [2.8.1](https://github.com/uias/Tabman/releases/tag/2.8.1) but this bug still exists in that version. Will also get the docs updated!

**Comment by Kashif-Rabbani** [comment]
@msaps Thank you so much. I am glad you are now able to reproduce it. Waiting for it to get resolved soon.

**Comment by msaps** [comment]
@Kashif-Rabbani @zaidqattan resolved with #505 🎉
